contrib/packaging/docker/Dockerfile.runtime

```diff
@@ -41,35 +48,37 @@ apt-get install -y --no-install-recommends make git curl ca-certificates xz-util
 # Additional iproute2 build dependencies
   gcc git pkg-config bison flex build-essential \
 # Additional bpftool dependencies
-  python3 && \
+  python3
 #
 # iproute2
 #
+RUN \
```

Comment:
What's the reason for this change? Each `RUN` instruction will add a new layer and increase image size.

Reply:
it's in the PR description:
>add more RUN statements so the whole package does not have to be
rebuilt when fixing individual build stages

Reply:
Thanks for the pointer, somehow overlooked this. So then it's mainly a trade-off between incremental build speed and image size. Do we in practice need to rebuild often while fixing individual build stages (outside of this PR)?